autonomous/forward.py

```diff
@@ -0,0 +1,14 @@
+from magicbot import AutonomousStateMachine, state
+from components.drivetrain import Drivetrain
+
+
+class Forward(AutonomousStateMachine):
+    MODE_NAME = "Forward"
+    DEFAULT = True
```

Comment:
So the constants won instead of the the Enums, eh?

Reply:
Robotpy needs those for the autonomous selector, I'm still somewhat undecided about the rest. Constants just don't tell you if they belong in groups, but that can remedied with lots of comments, so I think we'll probably be using constants for now.